nightly-benchmark/run-dgx-benchmark.sh

```diff
@@ -21,7 +21,7 @@ which python
 
 echo "Cythonize Distributed"
 pushd "${CONDA_PREFIX}/lib/python3.8/site-packages/"
-cythonize -i \
+cythonize -i --directive="profile=True" \
```

Comment:
This is explained in the [Cython docs for profiling]( https://cython.readthedocs.io/en/latest/src/tutorial/profiling_tutorial.html#enabling-profiling-for-a-complete-source-file ).